tests/Fixtures/inc/Engine/Cache/PurgeActionsSubscriber/maybePurgeCacheOnTermChange.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+return [
+	'vfs_dir'   => 'wp-content/cache/wp-rocket/',
+
+	// Virtual filesystem structure.
+	'structure' => [
+		'wp-content' => [
+			'cache' => [
+				'wp-rocket' => [
+					'example.org'                                => [
+						'index.html'      => '',
+						'index.html_gzip' => '',
+					],
+					'example.org-wpmedia-594d03f6ae698691165999' => [
+						'index.html'      => '',
+						'index.html_gzip' => '',
+					],
+				],
+			],
+		],
+	],
```

Comment:
Could you use the default structure instead? 

Why? I'm thinking it would be good to exercise our code in a broader sense of our filesystem structure. 

If yes, then you can delete the entire element as the vfs test case will automatically pull in the default structure.

Reply:
I updated the tests to use the default structure & the new methods to perform the assertions. Can you check I used them correctly?